iphone/Classes/WebFont.m

```diff
@@ -151,9 +151,9 @@ -(UIFont *) font
                 }
             }
             if (font == nil) {
-                //NO valid family specified. Just check for characteristics. Semi bold is ignored here.
+                //NO valid family specified. Just check for characteristics.
                 if (self.isBoldWeight) {
-                    UIFont* theFont = [UIFont boldSystemFontOfSize:self.size];
+                    UIFont* theFont = [UIFont systemFontOfSize:self.size weight:UIFontWeightBold];
```

Comment:
@hansemannn  We could get it in before 7.0.0 but I am ok with pushing it out of 6.2.0 with a decent comment.